packages/blob-reader/helper.js

```diff
@@ -1,19 +1,24 @@
-import $ from 'jquery';
+function $(selector, rootElement = document) {
+  return rootElement.querySelector(selector);
+}
 
-function getBlobCodeInner(el) {
-  return [].slice.call(el.getElementsByClassName('blob-code-inner'));
+function $$(selector, rootElement = document) {
+  return [...rootElement.querySelectorAll(selector)];
 }
 
-function getBlobWrapper(rootElement = document) {
-  const ret = [
-    ...[].slice.call(rootElement.getElementsByClassName('blob-wrapper')),
-    ...[].slice.call(rootElement.getElementsByClassName('js-blob-wrapper')),
-    ...[].slice.call(
-      rootElement.querySelectorAll('[class*="highlight-source-"]'),
-    ),
-  ];
+function getBlobCodeInner(el) {
+  return $$('.blob-code-inner');
```

Comment:
Should `el` be passed to `$$()`?

Reply:
The perils of _not_ using every lint rule available 😃